pages/support/index.js

```diff
@@ -0,0 +1,125 @@
+import React from "react";
+import { NextSeo } from "next-seo";
+import { getDocsListBySection, getNavigation } from "~/services";
+import { Box, Icon, theme, styled } from "@washingtonpost/wpds-ui-kit";
+import ChevronRight from "@washingtonpost/wpds-assets/asset/chevron-right";
+import { Header } from "~/components/Markdown/Components/headers";
+import Link from "~/components/Markdown/Components/link";
+import { P } from "~/components/Markdown/Styling";
+
+const Masonry = styled("section", {
+  width: "100%",
+  display: "grid",
+  gridTemplateColumns: "repeat( auto-fit, minmax(250px, 1fr) )",
+  gridGap: "$100",
+  "@sm": {
+    gridTemplateColumns: "1fr",
+  },
+});
+
+// create a divider
+const Divider = styled("hr", {
+  gridColumnEnd: "span 2",
+  margin: "$050 0 $050",
+  padding: 0,
+  border: 0,
+  height: "1px",
+  backgroundColor: "$subtle",
+});
+
+const ChevronForLink = styled(ChevronRight, {
```

Comment:
This is the third time I see this `ChevronForLink` being created. Always the same. Could we pull it out and reuse it?

Reply:
lol I just took it out! I didn't write this page, but it wasn't being used 🤷